### ISSUE TITLE
'ranges' option does not exist in type 'ILPConfiguration'

### ISSUE THREAD

**Issue by sucy6330133** [wakirin/Litepicker#191]
**Describe the bug**
Our project is using typescript. When specify the ranges option with custom range, a compile error has been received.

```
Argument of type '{ element: HTMLElement; singleMode: false; position: string; firstDay: number; mobileFriendly: true; numberOfColumns: number; numberOfMonths: number; delimiter: string; plugins: string[]; setup: (picker: Litepicker) => void; ranges: Object; } is not assignable to parameter of type 'ILPConfiguration'.
Object literal may only specify known properties, and 'ranges' does not exist in type 'ILPConfiguration'. TS2345
```

**Comment by wakirin** [comment]
Will be added in the next release.
Thanks.

**Comment by wakirin** [comment]
fixed in 2.0.7
